src/core/layout/qgslayoutexporter.h

```diff
@@ -231,6 +235,14 @@ class CORE_EXPORT QgsLayoutExporter
        */
       QVector<qreal> predefinedMapScales;
 
+
+      /**
+       * Image quality, typically used for JPEG compression (whose quality ranges from 0 to 100)
```

Comment:
I'm not sure which formats would accept 0. For JPEG the minimum is 1

Reply:
Not sure either. I set the minimum to 1.